code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -102,6 +102,8 @@
 	var/list/food_effects = list()
```

Comment:
can you make this null here and initialize it in New near the top

Reply:
tried to do this and it broke all food effects